lib/logstash/outputs/rabbitmq.rb

```diff
@@ -33,15 +36,19 @@ class RabbitMQ < LogStash::Outputs::Base
       # Should RabbitMQ persist messages to disk?
       config :persistent, :validate => :boolean, :default => true
 
+      def initialize(params)
+        params["codec"] = "json" if !params["codec"]
+        super
+      end
+
```

Comment:
I didn't know you could do that! (I cribbed that code from the old plugin code). Will fix!